src/main/java/command/CommandCentre.java

```diff
@@ -0,0 +1,53 @@
+package command;
+
+import exception.InvalidCommandException;
+import utils.Ui;
+
+import java.util.HashMap;
+
+public class CommandCentre {
+
+    /**
+     * A collection to store and map command names to their
+     * respective Command.
+     */
+    private HashMap<String, Command> commands;
```

Comment:
I like how you use hashmap to represent commands which improves the efficiency of access.